app/assets/javascripts/views/AboutView.js

```diff
@@ -0,0 +1,37 @@
+define([
+  'jquery',
+  'backbone',
+], function($,Backbone) {
+
+  'use strict';
+
+  var AboutView = Backbone.View.extend({
+
+    el: '.content-static',
+
+    events: {
+      'click .downloadDoc': 'onDownloadDoc'
+    },
+
+    model: new (Backbone.Model.extend()),
```

Comment:
You are probably right :D but I was having an error and didn't why, so I just added a few random things to see if it would make it work :D Will remove it.